sinks/postgres/sinks.go

```diff
@@ -173,19 +173,21 @@ func (s sinksRepository) RetrieveAllByOwnerID(ctx context.Context, owner string,
 		"tags":        tags,
 	}
 	rows, err := s.db.NamedQueryContext(ctx, q, params)
+	s.logger.Debug("debugging query result", zap.Any("rows", rows), zap.Error(err))
 	if err != nil {
 		return sinks.Page{}, errors.Wrap(errors.ErrSelectEntity, err)
 	}
 	defer rows.Close()
 
 	var items []sinks.Sink
 	for rows.Next() {
-		dbSink := dbSink{MFOwnerID: owner}
-		if err := rows.StructScan(&dbSink); err != nil {
+		d := dbSink{MFOwnerID: owner}
+		s.logger.Debug("iterating over rows", zap.Any("sink", d))
```

Comment:
we have to be careful here - I believe we have "debug" logging in orb.live at the moment. this should probably be removed unless it's really necessary.

Reply:
yup, I will start adding TODOs in every debug log I add. I added those for local development, also, I don't think we need debug log in orb.live. is any debug log, really necessary in orb.live so we could check if we could go for info in those specific logs?

Reply:
i agree it would be better to make sure we are not on debug log level on orb.live. let's sync with @etaques

Reply:
Commit with changes
https://github.com/orb-community/orb/pull/2268/commits/27add81715582d191df00103ad74607cef7d68e0